src/main/java/com/softserve/security/handler/ArticleResponseException.java

```diff
@@ -0,0 +1,25 @@
+package com.softserve.security.handler;
+
+import com.softserve.exceptions.ArticleException;
+import com.softserve.exceptions.CustomException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+public class ArticleResponseException extends ResponseEntityExceptionHandler {
+
+    @ResponseStatus(value = HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(value
+            = { ArticleException.class})
+    protected ResponseEntity<Object> handleConflict(
+            RuntimeException ex, WebRequest request) {
+        String bodyOfResponse = "This should be application specific";
```

Comment:
can we set this String as a static constant from a static class?

Since you are repeating also in RestResponseEntityExceptionHandler class

Reply:
Oooops!!, this String is not used for important interactions, so, I would deleted. 
The reason is cause the Error handler resolve error message body for frontend.